main/src/main/scala/org/clulab/processors/clu/CluProcessor.scala

```diff
@@ -759,6 +760,7 @@ class CluProcessor protected (
     if(doc.sentences.length > 0) {
       assert(doc.sentences(0).tags.nonEmpty)
       assert(doc.sentences(0).entities.nonEmpty)
+      assert(doc.sentences(0).lemmas.nonEmpty)
```

Comment:
This is a place where a spot check will not work if certain (long) sentences will not be processed.

Reply:
`CluProcessor` will be deleted.